src/lib/setup_sharify.js

```diff
@@ -73,7 +73,6 @@ sharify.data = _.extend(
     "INTERCOM_SELLER_APP_ID",
     "INTERCOM_SELLER_ENABLED",
     "INTERCOM_BUYER_APP_ID",
-    "INTERCOM_BUYER_APP_SECRET",
```

Comment:
This was my bad...this should _not_ be included in sharify since it's a secret which is available clientside. It should be directly imported/required instead. Whoops! 😅